.github/workflows/delete-unused-canary-packages.yml

```diff
@@ -0,0 +1,114 @@
+            global-phrases-block
+            ads-block
+            alert-bar-block
+            article-body-block
+            alert-bar-content-source-block
+            article-tag-block
+            author-bio-block
+            byline-block
+            card-list-block
+            date-block
+            default-output-block
+            double-chain-block
+            extra-large-manual-promo-block
+            extra-large-promo-block
+            footer-block
+            full-author-bio-block
+            gallery-block
+            header-block
+            header-nav-chain-block
+            headline-block
+      - name: Delete many package oldest canaries pt 2
+        uses: JackHowa/delete-github-package-versions@v0.4.9
+        with:
+          version-pattern: "canary"
+          keep: 5
+          dry-run: false
+          version-query-order: "first"
+          # 20 packages max
+          names: |
+            htmlbox-block
+            large-manual-promo-block
+            large-promo-block
+            lead-art-block
+            links-bar-block
+            masthead-block
+            medium-manual-promo-block
+            medium-promo-block
+            numbered-list-block
+            overline-block
+            quad-chain-block
+            related-content-content-source-block
+            results-list-block
+            right-rail-block
+            right-rail-advanced-block
+            search-results-list-block
+            section-title-block
+            share-bar-block
+            shared-styles
+            simple-list-block
+      - name: Delete many package oldest canaries pt 3
+        uses: JackHowa/delete-github-package-versions@v0.4.9
+        with:
+          version-pattern: "canary"
+          keep: 5
+          dry-run: false
+          version-query-order: "first"
+          # 20 packages max
+          names: |
+            single-chain-block
+            small-manual-promo-block
+            small-promo-block
+            subheadline-block
+            tag-title-block
+            text-output-block
+            textfile-block
+            top-table-list-block
+            triple-chain-block
+            video-player-block
+            video-promo-block
+            content-api-source-block
+            collections-content-source-block
+            author-content-source-block
+            search-content-source-block
+            site-hierarchy-content-block
+            story-feed-author-content-source-block
+            story-feed-query-content-source-block
+            story-feed-sections-content-source-block
+            story-feed-tag-content-source-block
+      - name: Delete many package oldest canaries pt 4
+        uses: JackHowa/delete-github-package-versions@v0.4.9
+        with:
+          version-pattern: "canary"
+          keep: 5
+          dry-run: false
+          version-query-order: "first"
+          # 20 packages max
+          names: |
+            tags-content-source-block
```

Comment:
So it looks like we have to add every block individually - can we add something into the documentation that upon creation of a new block, we should be adding it into this delete method?

Reply:
good call